services/src/layers/layer.rs

```diff
@@ -67,12 +66,15 @@ pub struct LayerDefinition {
     pub symbology: Option<Symbology>,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
 pub struct LayerCollection {
-    id: LayerCollectionId,
-    name: String,
-    description: String,
-    items: Vec<CollectionItem>,
+    pub id: ProviderLayerCollectionId,
+    pub name: String,
+    pub description: String,
+    pub items: Vec<CollectionItem>,
+    pub entry_label: Option<String>, // a common label for the collection's entries, if there is any // TODO: separate labels for collections and layers?
+    pub properties: Vec<(String, String)>,
```

Comment:
Why did you move this?

Reply:
it makes much more sense this way, because they belong to the collection that is currently being listed. When I render a collection in the UI, I want to print the properties and I want to display the entryLabel. Before the change I had to take the information from the parent collection's corresponding child entry.